packages/components/src/tooltip/test/index.js

```diff
@@ -11,60 +11,55 @@ import Tooltip from '../';
  * WordPress dependencies
  */
 import { TOOLTIP_DELAY } from '../index.js';
-import { act } from '@testing-library/react';
 
 describe( 'Tooltip', () => {
 	describe( '#render()', () => {
 		it( 'should render children (abort) if multiple children passed', () => {
-			// Mount: Enzyme shallow does not support wrapping multiple nodes.
-			const wrapper = mount(
+			const { container } = render(
 				<Tooltip>
 					<div />
 					<div />
 				</Tooltip>
 			);
 
-			expect( wrapper.children() ).toHaveLength( 2 );
+			expect( container.children ).toHaveLength( 2 );
 		} );
 
 		it( 'should render children', () => {
-			const wrapper = shallow(
+			render(
 				<Tooltip position="bottom right" text="Help text">
 					<button>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-			expect( wrapper.type() ).toBe( 'button' );
-			expect( button.children() ).toHaveLength( 1 );
-			expect( button.childAt( 0 ).text() ).toBe( 'Hover Me!' );
+			const button = screen.getByRole( 'button' );
+			expect( button.nodeName ).toBe( 'BUTTON' );
+			expect( button.childNodes ).toHaveLength( 1 );
+			expect( button ).toHaveTextContent( 'Hover Me!' );
 		} );
 
 		it( 'should render children with additional popover when over', () => {
-			const wrapper = shallow(
+			const { container } = render(
 				<Tooltip position="bottom right" text="Help text">
 					<button>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const event = { type: 'focus', currentTarget: {} };
-			wrapper.simulate( 'focus', event );
+			const button = screen.getByRole( 'button' );
+			button.focus();
+			expect( button.nodeName ).toBe( 'BUTTON' );
```

Comment:
I did the same mistake a few days ago. Here's the recommended way to go around these: https://github.com/WordPress/gutenberg/pull/42981#discussion_r938850094

Reply:
Fixed by cdafb54 👍